akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -89,10 +186,10 @@ private[akka] object StreamLayout {
         attributes)
     }
 
-    def grow(that: Module): Module = grow(that, Keep.left)
+    def compose(that: Module): Module = compose(that, Keep.left)
 
-    def grow[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
-      if (Debug) validate()
+    def compose[A, B, C](that: Module, f: (A, B) ⇒ C): Module = {
```

Comment:
minor (i.e. not a blocker for merging this): other methods got scaladoc, would be nice if those got it too.